src/utils/nextRunAt.ts

```diff
@@ -28,21 +29,30 @@ export function isValidHumanInterval(value: unknown): value is string {
 export const computeFromInterval = (attrs: IJobParameters): Date => {
 	const previousNextRunAt = attrs.nextRunAt || new Date();
 	log('[%s:%s] computing next run via interval [%s]', attrs.name, attrs._id, attrs.repeatInterval);
+
 	const lastRun = dateForTimezone(attrs.lastRunAt || new Date(), attrs.repeatTimezone);
+
+	const cronOptions: ArgumentsType<typeof parseExpression>[1] = {
```

Comment:
no need for a custom "ArguementTypes"; just use the build in versino "Parameters".
https://www.typescriptlang.org/docs/handbook/utility-types.html#parameterstype